ocaml/xapi/quicktest_raw_vdi_import.ml

```diff
@@ -0,0 +1,55 @@
+(*
```

Comment:
Hi, Gabor. Thanks for your review.
Updated the file name, please help to merge if there are no other comments.
Thanks in advance.